### PR TITLE
JFrog updates, v1 draft, DOC-3625

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request

- Preview link: [Jfrog Artifact and JFrog xray scanner reference](https://64e8ed2cd561de2ae606a01f--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference/)

### CODE REVIEW

**Review by r0610205 — DISMISSED** [review]

File: docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference.md [hunk 1]
```diff
@@ -1,35 +1,28 @@
 ---
-title: Jfrog Xray scanner reference
-description: Image scans with Jfrog Xray
+title: Jfrog Artifact and JFrog xray scanner reference
```
Comment (michaelcretzman) [inline docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference.md:2]:
```suggestion
title: JFrog Artifactory and JFrog Xray scanner reference
```

File: docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference.md [hunk 2]
```diff
@@ -1,35 +1,28 @@
...
+description: Image scans with Jfrog xray
 sidebar_position: 330
 ---
 
-You can set up Jfrog Xray scans using a Security step: create a CI Build or Security Tests stage, add a Security step, and then add the `setting:value` pairs as specified below.
+You can ingest scan results from JFrog Artifact and Jfrog xray. The standard workflow is to create a CI Build or Security Tests stage to your pipeline, add a Security step, and then use `setting:value` pairs to configure the step as specified below.
```
Comment (michaelcretzman) [inline docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference.md:7]:
```suggestion
You can ingest scan results from JFrog Artifactory and JFrog Xray. The standard workflow is to create a CI Build or Security Tests stage in your pipeline, add a Security step, and then use `setting:value` pairs to configure the step as specified below.
```

File: docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference.md [hunk 3]
```diff
@@ -1,35 +1,28 @@
...
-import StoDinDRequirements from '/docs/security-testing-orchestration/sto-techref-category/shared/dind-bg-step.md';
-```
-
-<StoDinDRequirements />
-
--->
+- Harness STO supports `ingestionOnly` scans with Jfrog Xray. `orchestrationOnly` and `dataLoad` scans are not supported. 
 
-### Root access requirements
+- You need to run the scan step with root access if you need to add trusted certificates to your scan images at runtime.
```
Comment (michaelcretzman) [inline docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference.md:15]:
```suggestion
- If you need to add trusted certificates to your scan images at runtime, then you need to run the scan step with root access. 
```

Comment (michaelcretzman) [reply docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference.md:15]: dependent clause goes 2nd

File: docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference.md (hunk 1 above)
Comment (michaelcretzman) [reply docs/security-testing-orchestration/sto-techref-category/xray-scanner-reference.md:2]: these are product names

**Review by michaelcretzman — APPROVED** [review]
Approved with suggestions and fixes.

**Comment by douglas-j-bothwell** [comment]
CleanShot 2023-08-28 at 15 24 03

**Comment by douglas-j-bothwell** [comment]
Rajiv approval here: https://harness.atlassian.net/browse/DOC-3625?focusedCommentId=562170

**Pull request merged**